docker-bake.hcl

```diff
@@ -0,0 +1,324 @@
+# Builds for Noteable kernels are managed by https://docs.docker.com/engine/reference/commandline/buildx_bake/
```

Comment:
thanks for the update; is there a way to break this up into separate files? looks like it might get unwieldy over time

Reply:
It can be separated into multiple files, and those files targeted with `-f` similar to Dockerfile. But I don't think there's any inheritance between files, so it would break up the build targets themselves, but we'd need to replicate things like variables, common labels, etc. 

I'll do some follow-up work to investigate the possibilities here